docs/_includes/Statements.md

```diff
@@ -431,20 +485,19 @@ For this form, the guards are evaluated in some undetermined order
 until one is found that is true, in which case the corresponding statements
 are executed. If none of the guards evaluates to true, then the
 loop execution is terminated.
+k
 
-### Loop Specifications
+## Loop Specifications {#sec-loop-specification}
 For some simple loops, such as those mentioned previously, Dafny can figure
 out what the loop is doing without more help. However, in general the user
```

Comment:
This is rare. I think it would be better to say "To reason about a loop, the user" (and then drop "more" on L493).